posts/2024-02-01-january-ecodev.mdx

```diff
@@ -0,0 +1,123 @@
+
+
+## Technical Releases
+
+**The Waku React package:** The package provides the components and UI adapters needed to integrate Waku SDK into React applications effortlessly. We developed the package in response to feedback from hackathon participants last year. The documentation, including detailed steps on using the package, [can be found here](https://docs.waku.org/guides/js-waku/use-waku-react/). 
+
+There is also a [beginner-friendly tutorial](https://blog.waku.org/tictactoe-tutorial/) for building a simple tic-tac-toe game that can be played by two players who are connected using Waku.
+
+**Open Bounties:** We have reviewed and revised the Waku bounties, ensuring that these tasks are beginner-friendly. Our goal is to make it easier for newcomers to Waku to contribute and, in turn, support the growth of the ecosystem.Anyone can work on the bounties at their own pace, as there is no hard cut-off date. When a bounty proposal is approved, it will be marked as ‘Assigned’. Find the open bountie [here.](https://github.com/waku-org/bounties/issues)
+
+
+## Team Updates
+
+The team has expanded this month, with the new EcoDev Team Lead Marina and a new Business Development Lead, Pedro, joining the team.
+
+You can connect with [Marina](https://twitter.com/Crypto_marina) and [Pedro](https://twitter.com/bigbraincap) on X.
+
+If you are interested in working with Waku, [we are currently hiring a Software Engineer](https://boards.greenhouse.io/waku/jobs/5607301) to work with the Waku Chat SDK. The position is fully remote, and we hire globally. 
+
+
+---
+
+
+## Waku Community Call and AMA
+
+Waku will host a Community Call on X on the last Friday of each month. The objective of the call is to update the community on our progress and provide a space where questions can be put to the team 
+
+The first Community Call took place last week on the 26th of January. You can [listen back to the full Space](https://twitter.com/Waku_org/status/1750927368644919722), and read the highlights below. 
+
+[Vaclav](https://twitter.com/vpavlin), [Guru](https://twitter.com/hackyguru), and [Aaron](https://twitter.com/vrycmfy) from the Waku team participated in the call. 
+
+
+### Waku network updates 
+
+**Vaclav**: If you don't know what the Waku Network is, it is our attempt to formalise a network of peer-to-peer nodes that would form some network running Waku protocols. We are very proud to have released the Gen 0, and there will likely be more generations to come right before Devconnect in Istanbul this November.
+
+It's just the beginning for the Waku Network; we're still dogfooding it. You can see [the list of milestones and the roadmap for improvements](https://github.com/waku-org/pm/issues/101) put forward by the research team. There’s also an interesting discussion you can join in the [Vac Forum](https://forum.vac.dev/t/message-validation-marketplace-waku-network-of-networks/262/2).
```

Comment:
I think instead you can say that we are drafting the roadmap to come for 2024 (subject to change): https://github.com/waku-org/pm/blob/master/ROADMAP.md the post gen 0 issue was more a discussion artifact.

Reply:
updated